### PR TITLE
Fixed template content

### PR DESCRIPTION
**Pull request by otani88**
Actual proxysql config:
https://github.com/sysown/proxysql/blob/master/src/proxysql.cfg

### CODE REVIEW

**Comment by otani88** [comment]
Fix version of proxysql, please.

**Comment by ernestas-poskus** [comment]
Thank you for the pull request, will look into version issue

**Comment by ernestas-poskus** [comment]
Version issue is fixed https://github.com/ernestas-poskus/chef-proxysql/pull/22

**Comment by otani88** [comment]
I see problems with installing mysql and another resources that don't depends on my changes.

File: templates/default/proxysql.cnf.erb [hunk 1]
```diff
@@ -18,17 +18,17 @@ mysql_servers=<%= to_cnf(@mysql_servers) %>
...
 <% if !@mysql_query_rules.empty? %>
 # Defines MySQL Query Rules
-mysql_query_rules=<%= to_cnf(@mysql_query_rules) %>
+mysql_query_rules:<%= to_cnf(@mysql_query_rules) %>
 <% end %>
 
 <% if !@schedulers.empty? %>
 # Defines schedulers
-schedulers=<%= to_cnf(@schedulers) %>
+scheduler=<%= to_cnf(@schedulers) %>
```
Comment (ernestas-poskus) [inline templates/default/proxysql.cnf.erb:31]: :+1:

File: templates/default/proxysql.cnf.erb [hunk 2]
```diff
@@ -18,17 +18,17 @@ mysql_servers=<%= to_cnf(@mysql_servers) %>
...
 <% if !@mysql_users.empty? %>
 # Defines all the MySQL users
-mysql_users=<%= to_cnf(@mysql_users) %>
+mysql_users:<%= to_cnf(@mysql_users) %>
 <% end %>
 
 <% if !@mysql_query_rules.empty? %>
 # Defines MySQL Query Rules
-mysql_query_rules=<%= to_cnf(@mysql_query_rules) %>
+mysql_query_rules:<%= to_cnf(@mysql_query_rules) %>
```
Comment (ernestas-poskus) [inline templates/default/proxysql.cnf.erb:26]: Isn't `=` and `:` are the same?

File: templates/default/proxysql.cnf.erb [hunk 3]
```diff
@@ -18,17 +18,17 @@ mysql_servers=<%= to_cnf(@mysql_servers) %>
 
 <% if !@mysql_users.empty? %>
 # Defines all the MySQL users
-mysql_users=<%= to_cnf(@mysql_users) %>
+mysql_users:<%= to_cnf(@mysql_users) %>
```
Comment (ernestas-poskus) [inline templates/default/proxysql.cnf.erb:21]: Isn't `=` and `:` are the same?

**Comment by ernestas-poskus** [comment]
According to http://www.hyperrealm.com/libconfig/libconfig_manual.html#Configuration-File-Grammar

```
setting = name (":" | "=") value (";" | "," | empty)
```

**Review by ernestas-poskus — APPROVED** [review]

File: templates/default/proxysql.cnf.erb (hunk 3 above)
Comment (otani88) [reply templates/default/proxysql.cnf.erb:21]: No, I checked it

**Comment by ernestas-poskus** [comment]
@mkryva 

```
             ================================================================================
             
             Chef::Exceptions::Package
             -------------------------
             No candidate version available for proxysql
             
             Cookbook Trace:
```

looks like, again, version is not available :man_facepalming:

**Comment by otani88** [comment]
Because 1.4.6-1. 
```proxysql/unknown 1.4.6-1.1.xenial amd64```

**Comment by ernestas-poskus** [comment]
@mkryva I debugged this issue and found that new Chef version 14 fails, so I locked Chef version.

Furthermore added special attribute `lock_version` which is false for kitchen tests so it will aid in easier development. Please rebase for the last time.

**Comment by ernestas-poskus** [comment]
@mkryva rebase please

**Comment by otani88** [comment]
Done

**Comment by ernestas-poskus** [comment]
Thank you for contribution :+1:

**Pull request merged**